packages/blade/.storybook/react/manager.js

```diff
@@ -8,6 +8,10 @@ import { INTERNAL_STORY_ADDON_PARAM } from './constants';
 
 const surafaceTextNormal = 'rgba(19, 38, 68, 1)';
 const bladePrimary = 'rgba(21, 102, 241, 1)';
+const bladeTextFont =
+  '"Inter", -apple-system, BlinkMacSystemFont, San Francisco, Segoe UI, Roboto, Helvetica Neue, sans-serif';
```

Comment:
can you also update storybook/installation guide? https://blade.razorpay.com/?path=/story/guides-installation--page#-add-blade-to-your-application

Reply:
Yup will update it in the end after other typography changes are done

Reply:
https://github.com/razorpay/blade/pull/1804#discussion_r1396831722

Reply:
add this to that umbrella todo as well

Reply:
https://github.com/razorpay/blade/issues/1852